actions/workflows/create_vm_role.yaml

```diff
@@ -28,16 +39,16 @@ st2cd.create_vm_role:
             input:
                 key: ami_<% $.distro %>
             on-success:
-                - check_cname_if_it_already_exists
```

Comment:
And if we want more explanation, we can always add comments at the top of the task so there's no need to make the task name unnecessary long...